Configuration and UI Adjustment

RakulAgn opened the pull request:
feat: Add configuration to prevent auto-formatting
- Prevents unwanted formatting changes during contributions
- Reduces noise in pull request diffs

fix: Adjust events area styling and improve readability
- Change padding from p-4 to p-6
- Replace "This Month" with actual month name
- Remove hover clamp to prevent UI shake
- Add text-nowrap to community text for better visibility

JustinBenito commented:
Hey @RakulAgn amazing descriptionof your PR, but it would be great if you could add screenshots and explain the code change a bit aswell. 
Thanks a lot
Cheers :)

RakulAgn commented:
Hey @JustinBenito its All about event card ui changes currently you can view in live site if you hover the card the it will shaking because of clamp class i have removed that and padding changed the corners are more near to the edges and changed the text this month to actually month name 

and added configuration for preventing code format if some one contributing it will make the PR ugly i hope you can check the code

JustinBenito commented:
Yup got that @RakulAgn 
But since you have made some UI changes,  it would be great if you could add a screenshot. 
There seems to be few merge conflicts aswell with events.tsx which I'll look into tomorrow as soon as possible and will update you. In the meanwhile it would be really great if you could add the screenshot of your update since its a UI change. 
Thanks a lot
Cheers :)

RakulAgn commented:
<img width="1440" alt="Screenshot 2025-01-04 at 7 53 06 AM" src="https://github.com/user-attachments/assets/85e9e926-6188-49ac-93cc-b20ad4baa757" />

I Hope this is helpful for you  @JustinBenito

I like to add few Contributions on the card container can i instead of wrapping everthing below we can add scroll horizontal and for events we can scrape the data instead of updating individualy for database you can use in-memory free database for that just on of the options if you like

JustinBenito commented:
Hey @RakulAgn Thanks a lot for adding that screenshot, It is great to see your update. 
But the issue of clamp on hover and text-nowrap is fixed with others pulling different PRs at different timelines. The padding change might be redundant since it doesn't add much to the UI and changing this month to name of the months is not something that we are looking at, at the moment. But nevertheless thanks a lot for your inputs and contributions. 
Cheers :)
